modules/ROOT/pages/netsuite/netsuite-openair-connector.adoc

```diff
@@ -36,31 +32,24 @@ The OpenAir connector is compatible with:
 [%header%autowidth.spread]
 |===
 |Application/Service|Version
-|Mule Runtime|EE 4.1.0 and later
+|Mule|EE 4.1.0 and later
 |Anypoint Studio|7.1 and later
 |Java|JDK 8
 |===
 
-== To Connect in Design Center
+== Connect in Flow Designer
 
-. Click a trigger such as HTTP Listener or Scheduler.
-. To create an HTTP global element for the connector, first set the path where your mule application listens:
-+
-image::netsuite/netsuite-openair-path.png[Path]
-+
-. Click Edit and set these fields:
-+
-image::netsuite/netsuite-openair-http.jpg[Trigger options]
-+
-[%header%autowidth.spread]
-|===
-|Field |Description
-|Protocol | Protocol selected for the HTTP endpoint, it can be HTTP or HTTPS (secure).
-|Host| IP address where your Mule application listens for requests.
-|Port| Port address where your Mule application listens for requests.
-|Base Path| An optional base path, the Mule application listens to the base path + path.
-|===
-+
+Flow Designer provides input source (triggers) for:
+
```

Comment:
Please reformat this list in the way that I have edited previous similar lists.
See the style reference.